include/boost/geometry/algorithms/detail/partition.hpp

```diff
@@ -408,17 +421,17 @@ public :
         typename OverlapsPolicy2,
         typename VisitBoxPolicy
     >
-    static inline void apply(Box const& box,
-            IteratorVector1 const& input1,
-            IteratorVector2 const& input2,
-            std::size_t level,
-            std::size_t min_elements,
-            VisitPolicy& visitor,
-            ExpandPolicy1 const& expand_policy1,
-            OverlapsPolicy1 const& overlaps_policy1,
-            ExpandPolicy2 const& expand_policy2,
-            OverlapsPolicy2 const& overlaps_policy2,
-            VisitBoxPolicy& box_policy)
+    static inline bool apply(Box const& box,
+                             IteratorVector1 const& input1,
+                             IteratorVector2 const& input2,
+                             std::size_t level,
+                             std::size_t min_elements,
+                             VisitPolicy& visitor,
+                             ExpandPolicy1 const& expand_policy1,
+                             OverlapsPolicy1 const& overlaps_policy1,
+                             ExpandPolicy2 const& expand_policy2,
+                             OverlapsPolicy2 const& overlaps_policy2,
+                             VisitBoxPolicy& box_policy)
```

Comment:
Next time, if possible, please don't mix layout changes with other changes. It makes the review more work.

About the layout change itself, this is how it is more and more done. But it is not how we started. We started by just intending 3 or 4 tabs (units 4 spaces). So independent on method name.

A drawback of this new convention is that, if the method name changes, or the return type (in length), all the lines have to change.

Another, larger, drawback is with long method names, it is all very intended, has to be split more (so into more lines), and in the end is less readable.

Especially because of the last reason, I don't know if I agree. However, sometimes I see myself taking it over also because QtCreator nearly forces it... that is maybe how this appears here too.